app/routes/verifier.py

```diff
@@ -284,6 +284,8 @@ async def accept_proof_request(
                     prover=prover,
                     presentation=presentation,
                 )
+            else:
```

Comment:
In this situation, we're only asserting that it's a valid prover if there is a connection id. I feel like a to-do should still remain here, so we have a way of asserting a valid prover even if there's no connection associated with the proof.

Reply:
I updated the log message. Not needed to resolve further